Framework/Lumberjack/CocoaLumberjack.h

```diff
@@ -0,0 +1,39 @@
+//
+//  CocoaLumberjack.h
+//  CocoaLumberjack
+//
+//  Created by Andrew Mackenzie-Ross on 3/02/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+//! Project version number for CocoaLumberjack.
+FOUNDATION_EXPORT double CocoaLumberjackVersionNumber;
+
+//! Project version string for CocoaLumberjack.
+FOUNDATION_EXPORT const unsigned char CocoaLumberjackVersionString[];
```

Comment:
A quick look at UIKit shows that `</>` seems to be preferred. If it's so we should maybe modify the original one and check if that breaks CocoaPods. Not making it breaks Carthage?